### PR TITLE
document openstack-magnum-k8s-image-x86_64, which

### PR DESCRIPTION
**Pull request by cjschroder**
provides the openstack-magnum-k8s-image.x86_64.qcow2 VM for OpenStack Magnum Container
    Orchestration

### CODE REVIEW

File: xml/book_cloud_suppl.xml [hunk 1]
```diff
@@ -1406,6 +1406,13 @@
       The <systemitem class="resource">python-magnumclient</systemitem> is installed.
      </para>
     </listitem>
+            <listitem>
+     <para>
+    Install the <package>openstack-magnum-k8s-image-x86_64</package> package. 
+    This provides a Kubernetes-based virtual machine, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>, that provides OpenStack Magnum Container
```
Comment (jgrassler) [inline xml/book_cloud_suppl.xml]: I've got a few modifications for this sentence (newly added stuff highlighted):

This *package* provides a virtual machine *image with Kubernetes pre-installed*, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>*. OpenStack Magnum uses this image when creating clusters with its `k8s_opensuse_v1` driver.*

File: xml/book_cloud_suppl.xml [hunk 2]
```diff
@@ -1406,6 +1406,13 @@
       The <systemitem class="resource">python-magnumclient</systemitem> is installed.
      </para>
     </listitem>
+            <listitem>
+     <para>
+    Install the <package>openstack-magnum-k8s-image-x86_64</package> package. 
+    This provides a Kubernetes-based virtual machine, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>, that provides OpenStack Magnum Container 
+    Orchestration.
```
Comment (sayalilunkad) [inline xml/book_cloud_suppl.xml]: This provides a SLES12SP2 based docker image, <filename>openstack-magnum-k8s-image.x86_64.qcow2</filename>, on which you can deploy kubernetes.

File: xml/book_cloud_suppl.xml (hunk 1 above)
Comment (sayalilunkad) [inline xml/book_cloud_suppl.xml]: I agree.

Comment (cjschroder) [inline xml/book_cloud_suppl.xml]: Metoo, thanks!

**Comment by cjschroder** [comment]
Ignore xml/depl_nodes.xml-- I have attached a PDF to make this easier to read. Please especially review the commands. The relevant section is '5.1 Deploying a Kubernetes Cluster from Command Line'

[openstack-magnum-package.pdf](https://github.com/SUSE-Cloud/doc-cloud/files/1142127/openstack-magnum-package.pdf)

**Comment by sayalilunkad** [comment]
needs rebase

**Comment by aclark801** [comment]
Does Flannel is mentioned in the first paragraph.  Does the magnum-k8s package include Flannel?  May want to note that.

**Comment by cjschroder** [comment]
> Does Flannel is mentioned in the first paragraph. Does the magnum-k8s package include Flannel? May want to note that.

It does not. I'll have a look at the whole supplement to see if it's mentioned somewhere else.

**Pull request merged**